content/en/docs/usage/kubectl-plugin.md

```diff
@@ -15,13 +15,21 @@ While the kubectl plugin is supported, it is recommended to use
 ## Installation
 You need the `kubectl-cert-manager.tar.gz` file for the platform you're using, these can be found on our [GitHub releases page](https://github.com/jetstack/cert-manager/releases).
 In order to use the kubectl plugin you need its binary to be accessible under the name `kubectl-cert_manager` in your `$PATH`.
+
+### macOS/Linux
 Run the following commands to set up the plugin:
 ```console
 $ curl -L -o kubectl-cert-manager.tar.gz https://github.com/jetstack/cert-manager/releases/latest/download/kubectl-cert_manager-linux-amd64.tar.gz
 $ tar xzf kubectl-cert-manager.tar.gz
 $ sudo mv kubectl-cert_manager /usr/local/bin
 ```
 
+### Windows
+1. Download the [latest version](https://github.com/jetstack/cert-manager/releases/latest/download/kubectl-cert_manager-windows-amd64.tar.gz).
```

Comment:
I need to double check, but I think `/latest` is unsuitable, because it redirects to what ever happens to be the newest release on the releases page, which might be alpha, beta, or patch release of a previous minor release.

Reply:
@wallrj `/latest` does not include the `pre-releases`, it actually points to the the one "tagged" `latest`.
![image](https://user-images.githubusercontent.com/11448520/144683456-803e2aaf-f72c-4380-9fad-13bbee29dc0f.png)
To make sure though I double checked it  with an old repo 😃 
The latest pre-release has a `test.txt` asset:
https://github.com/abousias/corepkgappveyor/releases/download/1.0.2-pre/test.txt
But it is not available in `/latest` because it gets redirected to `1.0.1`:
https://github.com/abousias/corepkgappveyor/releases/latest/download/test.txt